docs/index.md

```diff
@@ -37,11 +37,60 @@ Trestle provides tooling to help orchestrate the compliance process across a num
 
 ## Important Note:
 
-The current version of trestle supports NIST OSCAL 1.0.0-4.  There was a breaking change in OSCAL moving from
-version 1.0.0 to 1.0.2 mainly due to `prop` becoming `props` in AssessmentResults.  As a result, the current development path of trestle requires OSCAL 1.0.4, but for those who require OSCAL 1.0.0 please use trestle version 0.37.x.  That version is stable but will not have any features added, and we encourage users to move to OSCAL 1.0.4.
+The current version of trestle 3.x supports NIST OSCAL 1.1.2.
+Below shows trestle versions correspondence with OSCAL versions:
 
-OSCAL version 1.0.0 files are still handled on import but any AssessmentResults must conform to the OSCAL 1.0.4 schema, with
-props instead of prop.  And all files created by trestle will be output as OSCAL version 1.0.4.
+```
+trestle 3.x => OSCAL 1.1.2
+trestle 2.x => OSCAL 1.0.4
+trestle 1.x => OSCAL 1.0.2
```

Comment:
We can also add 
trestle 0.37.x => OSCAL 1.0.0

Reply:
ok